ptd_client_server/server.py

```diff
@@ -289,6 +330,13 @@ def handle_connection(self, p: lib.Proto) -> None:
         finally:
             self._ptd.stop()
 
+    def _set_range(self, volts_value: str, amps_value: str) -> None:
+        self._ptd.cmd(f"SR,V,{volts_value}")
+        self._ptd.cmd(f"SR,A,{amps_value}")
+        logging.info(f"Wait 10 seconds to apply setting for ampere and voltage")
```

Comment:
```suggestion
        logging.info("Wait 10 seconds to apply setting for amperage and voltage")
```

Reply:
I used "ampere and voltage" term from documentation.(SPEC-PTD-programmers-guide)

Reply:
OK, but please remove `f` then.

```suggestion
        logging.info("Wait 10 seconds to apply setting for ampere and voltage")
```